src/models/ERC721/ERC721Standard.js

```diff
@@ -89,6 +89,18 @@ class ERC721Standard extends IContract {
     this.params.contract.getContract().methods.setBaseURI(URI),
   );
 
+  /**
+   * Set Token URI
+   * @function
+   * @param {Object} params
+   * @param {string} params.tokenID
+   * @param {string} params.URI
+   * @return {Promise<*>}
+   */
+  setTokenURI = async ({ tokenID, URI }) => await this.__sendTx(
```

Comment:
use the method statment notation instead of property statment notation:

```
setTokenURI() {
  return ...
}
```

is not the same as 

```
setTokenURI = async ({ tokenID, URI }) => await this.__sendTx(
```

as the latter makes it not a prototyped method but instead a callable property.

We want to make those functions available in the prototype for N reasons, but scoping is a big one

Reply:
Makes sense, let´s open a bug/issue/update to do that for all objects